### DISCUSSION TITLE
Payment from GitHub sponsors

### DISCUSSION THREAD

**Discussion by ColinWater** [community/community#23091] [New to GitHub]
<p>Hi there, I created an account on GitHub which got a sponsor and later shut down. I have tracked my earnings on stripe and contacted you a week ago to schedule the payment of balance. Please respond to my query.</p>
<p>Ticket ID 1673432</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi there!</p>
<p>Thanks for reaching out! The Community Forum team is unable to respond to or escalate support tickets. Unfortunately, we can’t give a response time as it depends on the tickets ahead of you and the resourcing of our team. However, I did check in on this and your ticket has been received and is with the proper team. Appreciate your patience!</p>
<p>Lili</p>

**Comment by ColinWater** [comment]
<p>Ok then, I’ll wait for your respond. Hope it will be sooner.</p>

**Comment by kolumnin** [comment]
Can someone please help me understand if my sponsor configurations are appropriate or correct?
